### DISCUSSION TITLE
Hire Someone to Answer Q&A 🤷‍♂️

### DISCUSSION THREAD

**Discussion by DreamBagAttic** [keiwando/toonsquid#655] [Ideas]
Most of the questions in Q&A have gone unanswered the past few months. Maybe the answers are unknown, but hiring someone to give a knowledgeable response could really help beginning animators move forward. I love this software but it looks like when people get stuck they’re just kinda outta luck? I hope I’m wrong

**Comment by kavlor** [comment]
Probably easier said than done especially with TS 2 around the corner. Also you could look up Stayf Draws on Youtube , he seems to have  lots of Tutorials.

**Reply by budaloops** [reply]
It can be frustrating to attempt to create an animation only to spend the rest of the evening figuring out one small problem. 

However once we figure it out, it becomes second nature and we put ourselves are ahead of the curve because it’s likely that other users have encountered the same problems. I’d like to think that it would be in our best interest to then help those who encounter the problem that we once struggled with. By showing off our skills we position ourselves as experts in the eyes of those who are one or two steps behind and help them to realize their fullest “bouncing-ball” potential.

ToonSquid is still young and while there are relatively few members in this community (but steadily increasing) this is a great opportunity to become a master of the app and let our voices be heard, as we help to shape its growth. So let’s compare notes, and we’ll figure it out together!

For those of us who are some of the first to embrace ToonSquid as one of our go-to creative apps, It’s bonding time baby 😎

**Reply by kavlor** [reply]
Yes agreed. Also try and be clear about the problem and what you want to achieve , it might be people holding back helping because they are not sure how they can help.
